server/service/service_osquery.go

```diff
@@ -745,6 +745,14 @@ func (svc *Service) SubmitDistributedQueryResults(
 	}
 
 	if len(policyResults) > 0 {
+		if failingPolicies, err := svc.ds.NewFailingPoliciesForHost(ctx, host.ID, policyResults); err != nil {
```

Comment:
Eventually, I assume we'd just pass the policies registered in the AppConfig to `NewFailingPoliciesForHost`, so it doesn't waste time checking for irrelevant ones?

Reply:
Good idea!